src/status_im/init/core.cljs

```diff
@@ -21,13 +21,14 @@
 (fx/defn initialize-app-db
   "Initialize db to initial state"
   [{{:keys [view-id hardwallet initial-props desktop/desktop
-            supported-biometric-auth network/type]} :db}]
+            supported-biometric-auth network/type app-active-since now]} :db}]
```

Comment:
its not  in the db its in cofx , so you need to destructure it  `{{} :db now :now}`

Reply:
😢